pulp_rpm/app/models/custom_metadata.py

```diff
@@ -26,9 +26,10 @@ class RepoMetadataFile(Content):
     data_type = models.CharField(max_length=20)
     checksum_type = models.CharField(max_length=6)
     checksum = models.CharField(max_length=128)
+    relative_path = models.CharField(max_length=128)
```

Comment:
What is the thinking behind this length limit?
[relative_path](https://github.com/pulp/pulpcore/blob/master/pulpcore/app/models/content.py#L326) on the ContentArtifact model is a TextField with no limit, there is no guarantee that relative_path fits 128 chars.
I suggest to use TextField.